components/LandingPage.js

```diff
@@ -5,19 +5,19 @@ import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
-    <View style={styles.container}>
+    <View style={styles.container} testID="lp-view">
```

Comment:
This testID is unused. We could use it for unit testing, but I think snapshot will cover everything we'll need for now.

Reply:
I think we should take it out if it's not going to be used, but keep it if we will use it later.

Reply:
I'm always in favor of removing anything we are not using